src/Microsoft.Azure.ServiceBus/ExceptionReceivedEventArgsAction.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace Microsoft.Azure.ServiceBus
+{
+    using Microsoft.Azure.ServiceBus.Primitives;
+
+    /// <summary>Action taking place when <see cref="ExceptionReceivedEventArgs"/> is raised for a processed message.</summary>
```

Comment:
Okay, but meanwhile could you please remove the 'processed message' part? Incomplete documentation isn't as bad as wrong documentation :)